.gitignore

```diff
@@ -7,3 +7,9 @@ tabookey-gasless-0.1.0.tgz
 /.vscode/settings.json
 /.idea/
 /server/pkg/darwin_amd64/
+/serverdock/Contracts/
```

Comment:
remove these lines.
don't build in ./serverdoc. instead, build in ./build/serverdoc